src/domains/mapDomain.ml

```diff
@@ -58,19 +58,14 @@ end
 module type Groupable =
 sig
   include Printable.S
-  val classify: t -> int (* groups are sorted by this *)
-  val class_name: int -> string (* name of group *)
+  type group (* use [@@deriving show, enum] *)
+  val show_group: group -> string
+  val group_to_enum: group -> int
+  val group_of_enum: int -> group option
+  val to_group: t -> group
   val trace_enabled: bool
 end
 
-module StripClasses (G: Groupable) =
-struct
-  include G
-  let classify x = 0
-end
-
-(* Just a global hack for tracing individual variables. *)
```

Comment:
I've always wondered what this comment is referring to, but now I think it's about the `Groupable.trace_enabled`. Maybe it should just be moved there?

Reply:
Probably. Moved.